content/docs/guides/price-schedules.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Same Product, Multiple Price Schedules'
+section: 'Guides'
+priority: 6
+hidden: false
+---
+
+import { Link } from 'gatsby'
+
+Price schedules are one of the critical factors related to <Link to="main-concepts/product-visibility">product visibility</Link> and there are several ways that you can build relationships between products and price schedules.  One of the scenarios you may wish to achieve is one product with multiple price schedules.  _Why_ would you possibly want one product that had multiple different prices based on buyer/user?  Lets take a look at the example scenario below, and walk through a solution to achieve the desired result.
+
+> As a seller company, I sell two different memberships to my b2b ecommerce site.  The first is **Startup** and the second is **Enterprise**.  The **Enterprise** membership is considerably more expensive than the **Startup** membership, therefore I want to provide a slightly lower product pricing to my **Enterprise** customers.  Specifically, Jane Doe from Best Buy has signed up for an **Enterprise** membership, and John Deer from TechWorld has signed up for a **Startup** membership.
+
+How would we go about achieving this effect you might ask?  After reviewing the considerations, let's dive in!
+
+## Considerations
+
+1. In our example, we will be assigning price schedules to products and parties at the Buyer level, however you can get even more granular than that, and assign Product/Price Schedule combinations at the User Group level within a single Buyer.  This allows you further flexibility within your product visibility.
+
+2. This guide assumes that all products, categories and catalogs have been created successfully, with proper assignments.  Make sure that your product is assigned to a catalog that the buyer's users have access to, or you will have issues when trying to create your product assignments.
```

Comment:
Do we have docs, on how products, categories , and catalogs are created? I think it would be nice to have a link in here just a user didn't get that far.

Reply:
or maybe a quick comment on how to verify that a product has been assigned to a catalog. if they are still getting used to assignments?

Reply:
Done.